src/libasr/asr_utils.h

```diff
@@ -1266,15 +1266,15 @@ static inline std::string type_to_str_python(const ASR::ttype_t *t,
         }
         case ASR::ttypeType::Struct: {
             ASR::Struct_t* d = ASR::down_cast<ASR::Struct_t>(t);
-            return "struct " + std::string(symbol_name(d->m_derived_type));
+            return std::string(symbol_name(d->m_derived_type));
         }
         case ASR::ttypeType::Enum: {
             ASR::Enum_t* d = ASR::down_cast<ASR::Enum_t>(t);
-            return "enum " + std::string(symbol_name(d->m_enum_type));
+            return std::string(symbol_name(d->m_enum_type));
         }
         case ASR::ttypeType::Union: {
             ASR::Union_t* d = ASR::down_cast<ASR::Union_t>(t);
-            return "union " + std::string(symbol_name(d->m_union_type));
+            return std::string(symbol_name(d->m_union_type));
```

Comment:
I think this is not needed.

This was a feature added in https://github.com/lcompilers/lpython/commit/7f7dbde2f14a2ef244bd585f4dd69bc599efe8f5. `Struct`, `Enum`, `Union` are derived types. For them, printing just the symbol name does not seem sufficient (we also need their type information). 

For example, for `s: i32`, `type_to_str_python()` returns `i32` which makes clear that it is an integer/`i32`.
For `class S:`, `type_to_str_python()` returns just `S` which does not inform of the type of `S` (`Struct` or `Enum` or `Union`).

For other types, `type_to_str_python()` returns the type name, where as for derived types `type_to_str_python()` returns the symbol name (and not the type). Therefore, we had prepended the type information to derived types in https://github.com/lcompilers/lpython/commit/7f7dbde2f14a2ef244bd585f4dd69bc599efe8f5.